plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -717,12 +702,9 @@ public boolean accept(File pathname) {
 
     private KubernetesListBuilder generateAppResources(List<ImageConfiguration> images, EnricherManager enricherManager)
         throws IOException, MojoExecutionException {
-        Path composeFilePath = checkComposeConfig();
-        ComposeService composeUtil = new ComposeService(komposeBinDir, composeFilePath, log);
         try {
             File[] resourceFiles = KubernetesResourceUtil.listResourceFragments(realResourceDir);
-            File[] composeResourceFiles = composeUtil.convertToKubeFragments();
-            File[] allResources = ArrayUtils.addAll(resourceFiles, composeResourceFiles);
+            File[] allResources = ArrayUtils.addAll(resourceFiles);
```

Comment:
Why not delete this line and replace allResources with resourceFiles ? This line is virtually redundant now.

Reply:
Yes this line should be deleted, although probably the compiler is already doing it for you.